cli/tests/workers/http_worker.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+const body = new TextEncoder().encode("Hello World");
+for await (const conn of Deno.listen({ port: 80 })) {
```

Comment:
Try binding to some other port that is already used in tests, eg. 4500

Reply:
Turns out the culprit is TypeScript. It's taking time for the server to boot up before the request is made resulting in a failure. I switched the HTTP worker script to JS for now.